### PR TITLE
Tweak check-plugin / setup_runners.* / version-info scripts to work with new manifest.yml

### PR DESCRIPTION
**Pull request by peterzhuamazon**
*Issue #, if available:*
V301153385

*Description of changes:*
There are two parts of the changes in this PR:
1. Make check-plugin scripts work again after the manifest.yml changes
2. Make setup_runners more robust and generic in preparation for the new arch/platform
3. A new version-info.sh has been added to resolve the python issues across platforms

*Test Results:*
check-plugins: https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/467710814
setup_runners: https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/467499548

**Note: If this PR is related to Helm, please also update the README for related documentation changes. Thanks.**
**https://github.com/opendistro-for-elasticsearch/opendistro-build/blob/master/helm/README.md**

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: release-tools/scripts/releasenotes.sh [hunk 1]
```diff
@@ -22,8 +22,8 @@ set -e
 OLDIFS=$IFS
 REPO_ROOT=`git rev-parse --show-toplevel`
 ROOT=`dirname $(realpath $0)`;
-OD_VERSION=`python $REPO_ROOT/bin/version-info --od`
-ES_VERSION=`python $REPO_ROOT/bin/version-info --es`
+OD_VERSION=`python $REPO_ROOT/bin/version-info.sh --od`
+ES_VERSION=`python $REPO_ROOT/bin/version-info.sh --es`
```
Comment (gaiksaya) [inline release-tools/scripts/releasenotes.sh:26]: Why we need python for running shell file?

File: release-tools/scripts/version-info.sh [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+# Requirements:  Need to install yq (v4.0.0+) on your system:
+#                https://github.com/mikefarah/yq/releases/
+#
+# Starting Date: 2021-01-06
+# Modified Date: 2021-01-06
+###############################################################################################
+
+set -e
+ROOT=`dirname $(realpath $0)`;
+PLUGIN_LIST="$ROOT/manifest.yml"
```
Comment (gaiksaya) [inline release-tools/scripts/version-info.sh:24]: Can you call it rename variable to `manifest`. Plugin list is confusing in the code

File: release-tools/scripts/releasenotes.sh (hunk 1 above)
Comment (peterzhuamazon) [reply release-tools/scripts/releasenotes.sh:26]: missed

Comment (peterzhuamazon) [reply release-tools/scripts/releasenotes.sh]: fixed.

File: release-tools/scripts/version-info.sh (hunk 2 above)
Comment (peterzhuamazon) [reply release-tools/scripts/version-info.sh:24]: That is due to old file name is plugins.json, will change it.

Comment (peterzhuamazon) [reply release-tools/scripts/version-info.sh:24]: fixed.

Comment (gaiksaya) [reply release-tools/scripts/version-info.sh:24]: Not fixed here

**Review by gaiksaya — APPROVED** [review]

File: release-tools/scripts/version-info.sh [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+# Requirements:  Need to install yq (v4.0.0+) on your system:
+#                https://github.com/mikefarah/yq/releases/
+#
+# Starting Date: 2021-01-06
+# Modified Date: 2021-01-06
+###############################################################################################
+
+set -e
+ROOT=`dirname $(realpath $0)`;
+MANIFEST_FILE="$ROOT/manifest.yml"
```
Comment (sreekarjami) [inline release-tools/scripts/version-info.sh:24]: Can you change MANIFEST_FILE="./opendistro-build/release-tools/scripts/manifest.yml" , that will help us avoid realpath  command. This is in regard to running this script on the macos instances where the realpath is not present by default.

Comment (peterzhuamazon) [reply release-tools/scripts/version-info.sh:24]: The idea is if you move the scripts folder these script will still work, as long as they are in same folder
Hardcode it defeat the purpose here

**Review by sreekarjami — APPROVED** [review]

**Pull request merged**